Userland/Libraries/LibC/unistd.cpp

```diff
@@ -796,4 +796,10 @@ int pause()
 {
     return select(0, nullptr, nullptr, nullptr, nullptr);
 }
+
+int chroot(const char* path)
+{
+    dbgln("FIXME: chroot('{}')", path);
```

Comment:
```suggestion
    dbgln("FIXME: chroot(\"{}\")", path);
```

seeing `'not a single char'` would be strange :)